### PR TITLE
♻️. Clean-up for storybook and components

### PR DESCRIPTION
**Pull request by cedricengagor**
## PR Details

- Updated tooltip to zeroheight standards
- Updated help text component
	- Renamed to InlineHelp
	- Added Popover component
	- Updated structure for this to Tooltips/...
- Updated structure for buttons

### CODE REVIEW

File: src/Tooltips/Popover.test.tsx [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+
+    const inlineHelp = await renderer
+        .create(
+            <Popover content="Hello this is my awesome helptext!" visible={true} appendTo={element}>
+                <div>Popover Content</div>
+            </Popover>
+        )
+        .toJSON();
+
+    await expect(inlineHelp).toMatchSnapshot();
```
Comment (cedricengagor) [inline src/Tooltips/Popover.test.tsx:29]:
```suggestion
    const popover = await renderer
        .create(
            <Popover content="Hello this is my awesome helptext!" visible={true} appendTo={element}>
                <div>Popover Content</div>
            </Popover>
        )
        .toJSON();

    await expect(popover).toMatchSnapshot();
```

File: package.json [hunk 2]
```diff
@@ -7,14 +7,14 @@
...
-        "lint-fix": "eslint ./src --ext .jsx,.js,.ts,.tsx --cache --cache-strategy content --fix",
+        "lint": "eslint ./src --ext .jsx,.js,.ts,.tsx --cache --cache-strategy content --cache-location tmp/.eslintcache",
+        "lint-fix": "npm run lint -- --fix",
         "type-check": "tsc --noEmit -p tsconfig.json",
         "storybook": "start-storybook -p 6006",
         "build-storybook": "build-storybook",
         "storybook-docs": "start-storybook --docs",
         "build": "webpack --mode=production",
-        "prepublishOnly": "npm run build && npm run deploy-storybook",
+        "prepublishOnly": "npm run build",
```
Comment (cedricengagor) [inline package.json:17]:
```suggestion
        "prepublishOnly": "npm run build && npm run deploy-storybook",
```

**Review by thaoms — APPROVED** [review]
Doesn't moving everything remove all history?

File: src/Tooltips/Popover.tsx [hunk 3]
```diff
@@ -0,0 +1,63 @@
...
+            <Tippy
+                ref={ref}
+                placement={placement}
+                visible={visible}
+                appendTo={appendTo}
+                popperOptions={popperOptions}
+                render={
+                    (attrs) => {
+                        if (title) {
+                            title = <h2 className="text-base font-bold m-0 mb-8">{title}</h2>;
```
Comment (marcusrelacion) [inline src/Tooltips/Popover.tsx:46]: The color of the text in the popover/inline-help should be Charcoal-400. Other than that everything is looking good!

**Review by marcusrelacion — CHANGES_REQUESTED** [review]

**Comment by cedricengagor** [comment]
> Doesn't moving everything remove all history?

For me the history is maintained, so I guess it should be okay 👌

**Review by marcusrelacion — APPROVED** [review]
Looking good, thank you for updating the text color @cedricengagor!

**Comment by cedricengagor** [comment]
> Looking good, thank you for updating the text color @cedricengagor!

Thanks for the fast review @marcusrelacion, appreciate it 🙇

**Pull request merged**